doc/news/_preparation_next_release.md

```diff
@@ -49,7 +49,15 @@ The following section lists news about the [plugins](https://www.libelektra.org/
 - <<TODO>>
 - <<TODO>>
 
-### <<Plugin2>>
+### JNI
```

Comment:
Please put this as highlight, give more text about what changed, put links to jni/jna docu/tutorials, explain what was tested and what works now.

Reply:
updated